src/components/custom/_signup-public.js

```diff
@@ -255,244 +312,262 @@ const SignupPublic = ({
-                            external="true"
-                            href={deriv_app_url}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                        >
-                            <StyledHeader
-                                size="2.8rem"
-                                max_width="28.2rem"
-                                align="center"
-                                color="grey-8"
-                                mr="1.2rem"
-                            >
-                                {localize('Get a taste of the Deriv experience')}
-                            </StyledHeader>
-                            <img src={Arrow} alt="arrow desktop" />
-                        </LinkFlex>
-                    </BackgroundWrapper>
-                </Wrapper>
-            </Show.Desktop>
-            <Show.Mobile>
-                <MobileWrapper>
-                    <MobileSignupFormWrapper>
-                        <div>
-                            <StyledHeader type="section-title">
-                                {localize('Join over 1 million traders worldwide')}
-                            </StyledHeader>
-                            <br />
-                            <StyledHeaderText weight="bold" size="1.6rem">
-                                {localize('Sign up for your demo account now.')}
-                            </StyledHeaderText>
-                            <InputGroup>
-                                <InputWrapper>
-                                    <Input
-                                        id="email"
-                                        name="email"
-                                        type="text"
-                                        error={email_error_msg}
-                                        value={email}
-                                        background="white"
-                                        tabletBackground="green-1"
-                                        inputColor="grey-5"
-                                        inputBackground="grey-8"
-                                        labelFocusColor="grey-7"
-                                        labelColor="black-3"
-                                        label={localize('Email')}
-                                        placeholder={'example@mail.com'}
-                                        handleError={clearEmail}
-                                        onChange={handleInputChange}
-                                        onBlur={handleValidation}
-                                        autoFocus={autofocus}
-                                        autoComplete="off"
-                                        required
-                                        border="unset"
-                                        focusBorder="var(--color-grey-7)"
+        <SectionContainer>
+                <Show.Desktop>
+                    <Container>
```

Comment:
You can also extend this and set the max-width for to be say 500px for `tabletL` and below. If I'm not mistaken it also accept props to do so, will need to check. Think that it would make the form look better in tablet screen widths 😅

Reply:
Thanks @sean-binary , Made the changes in css